airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/build_image/steps/python_connectors.py

```diff
@@ -45,7 +45,7 @@ async def _create_builder_container(self, base_container: Container) -> Containe
         Returns:
             Container: The builder container, with installed dependencies.
         """
-        ONLY_BUILD_FILES = ["pyproject.toml", "poetry.lock", "poetry.toml", "setup.py", "requirements.txt"]
+        ONLY_BUILD_FILES = ["pyproject.toml", "poetry.lock", "poetry.toml", "setup.py", "requirements.txt", "README.md"]
```

Comment:
As the README.md file is declared in pyproject.toml the poetry check command checks if it's available. It's why we need it at build time.